test/functional/date/parse-date.js

```diff
@@ -122,10 +122,10 @@ QUnit.test( "should parse time presets", function( assert ) {
 	date.setSeconds( 7 );
 	date = startOf( date, "second" );
 	assertParseDate( assert, "5:35:07 PM", { time: "medium" }, date );
-	assertParseDate( assert, "٥،٣٥،٠٧ م", { time: "medium" }, date, ar );
+	assertParseDate( assert, "٥:٣٥:٠٧ م", { time: "medium" }, date, ar );
```

Comment:
In previous CLDR (when this test was created), `،` was used as time separator. In recent CLDR, it was reverted to use `:` again. Although, the parser was buggy and didn't actually check for that. It was still passing because whatever character was working there. After the fix, this needed fix.